README.md

```diff
@@ -25,7 +25,7 @@ brew upgrade
 
 After installing development tools, you'll need to set the password for MariaDB:
 ```
-sudo mysql_secure_installation
+sudo mariadb-secure-installation
```

Comment:
Running this says: 
```
mysql_secure_installation
/opt/homebrew/bin/mysql_secure_installation: Deprecated program name. It will be removed in a future release, use 'mariadb-secure-installation' instead
```